lerna.json

```diff
@@ -1,5 +1,13 @@
 {
   "packages": ["docs", "./packages/*"],
+  "command": {
+    "version": {
+      "allowBranch": "master",
+      "conventionalCommits": true,
+      "ignore": ["docs"],
+      "message": "build: publish"
+    }
+  },
```

Comment:
move the config here that i could. running through a yarn script or through the lerna cli should give us similar results